core/src/main/scala/com/spotify/featran/json/package.scala

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.featran
+
+import com.spotify.featran.transformers.{MDLRecord, WeightedLabel}
+import io.circe._
+import io.circe.parser._
+
+/**
+ * Package for an intermediate format for feature storage using JSON
+ */
+package object json {
```

Comment:
@richwhitjr I wonder if we can use the existent typeclass `JsonSerializable` to hold the serialization logic for the features. I think this way `FlatReader[String]` would just call encode and decode using the provide `JsonSerializable` typeclasses. wdyt?

Reply:
@richwhitjr let's do an enhancement afterwards ... issue #162